tests/auxil/networking.py

```diff
@@ -90,7 +91,7 @@ async def send_webhook_message(
     content_type: str = "application/json",
     get_method: Optional[str] = None,
     secret_token: Optional[str] = None,
-    unix: Optional[str] = None,
+    unix: Optional[Path] = None,
```

Comment:
maybe allow both str & Path and convert just to `Path(unix)`? We even have a type alias prepared for that somewhere.

Reply:
Does that matter for an internal testing method?